03-task-manager/starter/quizAnswers2.txt

```diff
@@ -0,0 +1,11 @@
+1. In our controllers we had repetitations with try and catch blocks. Asyncwrapper 
+basically helped us to clean up the clutter in the code.
+
+2.const getTask = asyncWrapper(async (req, res, next) => {
+    const {id: taskID} = req.params
+    const task = await Task.findOne({ _id: taskID })
+    if (!task){
+       return next(createCustomError("Not found!", 404));
```

Comment:
There are multiple ways but you I see that you'll are creating an error object in the next() call.   Another way is to throw the custom Error object and have that caught in the handler. In the handler, you will need first as err argument to catch that error and end req/res cycle. I think the latter one is neater and less  buggy.